porcelain/protocol.go

```diff
@@ -21,6 +22,7 @@ type protocolParamsPlumbing interface {
 	MessageQuery(ctx context.Context, optFrom, to address.Address, method string, params ...interface{}) ([][]byte, error)
 }
 
+// ProtocolParameters TODO(rosa)
```

Comment:
ProtocolParameters exposes certain commonly inspected parameters of the storage and retrieval protocols for use by the user and elsewhere in the application.